### DISCUSSION TITLE
Normalize the project version

### DISCUSSION THREAD

**Discussion by seapagan** [seapagan/fastapi-template#118] [Ideas]
This started as a small app for myself but has grown with many more features, and will continue to do so.

However, It is not yet (IMO) 'Version 1' quality. There are a few basics that still need to be added and things I'd like to change.

So, I'd like to 're-version' the library down to say `0.5.0`. This gives a lot of room to grow while still showing that the system is 'young'.

I'll leave this open for a week, and if there are no comments or good reasons not to, I'll do as above. 

Unsure whether to delete the previous releases or to re-version them. It is easier to upgrade derivatives of this project if you can take a diff from an existing release, so I'll most likely reversion and leave them.

@pdrivom comments?

**Comment by pdrivom** [comment]
Totally on board with the idea. Somethings must be fine tuned before considering it, v1 ready. I think it can start as `0.5.0` and it will grow from there. 

It would be better to create a version roadmap. The set of features included in each version and for each have a development environment setup and a production ready deploy option (LB/Reverse Proxy/..).

BTW [Github Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) can be a useful way to organise the project ToDos.

**Reply by seapagan** [reply]
> Totally on board with the idea. Somethings must be fine tuned before considering it, v1 ready. I think it can start as 0.5.0 and it will grow from there.

Yeah. This project started as a base for another project of mine, so I promoted it to V1 too soon. Now it's becoming more generic it needs to be re-versioned. 

> BTW [Github Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) can be a useful way to organise the project ToDos.

Yup, used this a few times. I'm just old-school and like the TODO.md file 🤣. As this project gets bigger then for sure we'll use GitHub Projects.

**Comment by seapagan** [comment]
I have done this. I chose to start at `0.4.0` just to be awkward :stuck_out_tongue_closed_eyes:
